run.py

```diff
@@ -10,7 +12,7 @@
 from hdx.hdx_configuration import Configuration
 from hdx.location.country import Country
 from hdx.utilities.easy_logging import setup_logging
-from hdx.utilities.path import script_dir_plus_file
+from hdx.utilities.path import script_dir_plus_file, temp_dir, get_temp_dir
```

Comment:
I think a couple of these imports are not used

Reply:
Thx, fixed